cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/graph/declarations/ValueDeclaration.kt

```diff
@@ -61,6 +61,8 @@ abstract class ValueDeclaration : Declaration(), HasType {
             }
         }
 
+    var aliases = mutableSetOf<Node>()
```

Comment:
Can we somehow restrict this into set of specific nodes? Does it always contain declarations or references? Or both?

Does it make sense to have an `HasAliases` that both references and declarations implement.

Reply:
Yes, makes sense

Reply:
Actually I think it always contains declarations but it could be potentially also something different